### ISSUE TITLE
Find: enter and shift+enter should not toggle field state

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#92510]
Issue Type: <b>Bug</b>

Maybe I'm getting something wrong, or the orca is giving me wrong information about the shortcut keys.

Steps to reproduce:
1. Activate orca.
2. Open a non empty file in vscode.
3. Press ctrl+f and type some text.
Note that orca announces the line found
4. Press the enter key.
Vscode jumps to the next line that contains the text typed in the step 3 and orca announces the line.
5. Press shift+enter.
Vscode jumps to the previous line as expected.
6. press tab.
The focus is moved to the match case field.
7. Press enter.
Instead of go to the next line, vscode toggle the status of The  the match case field.
The same happens if I press the shift+enter key.
The behavior appears to be the same for all fields, except for the field in which I type the text to be searched.
@isidorn I think the status of the check box should be changed with the space key and the enter and shift + enter keys should act as shortcuts to next mat and previous mat.

VS Code version: Code - Insiders 1.44.0-insider (ef75c4bb445e5021065cf0012cfa2c66e55bfa16, 2020-03-11T09:19:54.444Z)
OS version: Linux x64 5.5.8-arch1-1

**Comment by isidorn** [comment]
Great feedback and I agree. Renaming title to make it more concise.
Forwarding to @rebornix as he owns the find widget 
@rebornix can you change the when context for the enter and shift+enter to be whenever the focus is in the find, not only in the find input box. That should fix this?

**Comment by isidorn** [comment]
@rebornix assigning this to June and myself so I look into this.
Can you maybe provide some code pointers? Thanks

**Comment by isidorn** [comment]
@jvesouza thanks for feedback but now I thought a bit more about this, and we always trigger actions via Enter, so I would personaly not change this to be aligned with actions and toggles everywhere else.
`Shift+enter` makes sense that it should not be eaten by the find widget checkboxes.

So leaving this up to @rebornix to decide what to do here

**Comment by VSCodeTriageBot** [comment]
The described behavior is how it is expected to work. If you disagree, please explain what is expected and what is not in more detail. See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
